src/test/Database.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+<<<<<<<< HEAD:src/Database/Database.php
+namespace Musicplayer\Database;
+========
+namespace Musicplayer\test;
+>>>>>>>> 35fc4ab15355572795363c39b93c74dd77c61977:src/test/Database.php
+use PDO;
+
+class Database
+
+{
+    private const HOST = 'db';
+    private const DB = 'musicplayer';
+    private const CHARSET = 'utf8mb4';
+    private const DSN = 'mysql:host=' . self::HOST
+    . ';dbname=' . self::DB . ';charset=' . self::CHARSET;
+    private const USER = 'root';
+    private const PASSWORD = 'password';
+
+    private PDO $pdo;
+
+    public function __construct()
+    {
+        $options = [
+            PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+            PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION
+        ];
+
+        try {
+            $this->pdo = new PDO(self::DSN, self::USER, self::PASSWORD, $options);
+        } catch (\PDOException $e) {
+            echo '<p>There was an error connecting to the db</p>';
```

Comment:
I think this has overwritten what we did in story 2 - there was a throw new error here.